gematria/experiments/access_pattern_bm/stl_assoc_container_test.cc

```diff
@@ -81,9 +82,11 @@ void BM_STLAssocContainer_Flush(benchmark::State &state) {
   // Create a random associative container.
   auto container = CreateRandomSTLAssocContainer<Container>(size);
 
-  int sum = 0;
   for (auto _ : state) {
+    int sum = 0;
```

Comment:
With this change, we don't need the `sum = 0;` on line 98 anymore.

(and for some reason GitHub doesn't let me comment directly on line 98).